client/src/app/VariantTesting.res

```diff
@@ -30,14 +28,9 @@ let availableAdminVariants: list<variantTest> = list{NgrokVariant}
 let activeCSSClasses = (m: model): string =>
   m.tests |> List.map(~f=toCSSClass) |> String.join(~sep=" ")
 
-let apiRoot = (m: model): string => {
-  let useFSharp = List.member(~value=FsharpBackend, m.tests)
-  if useFSharp {
-    "/api-testing-fsharp/"
-  } else {
-    "/api/"
-  }
-}
+// todo: remove the input here
+let apiRoot = (_m: model): string => "/api/"
```

Comment:
Oh I forgot to do this

Reply:
decided to leave it in with commentary

Reply:
Makes sense. While I was at it, I remove it from the fetchContext; also felt like something we could add back in if need be, later.